src/model/guild/guild_id.rs

```diff
@@ -161,8 +161,9 @@ impl GuildId {
         builder.execute(http, self, user_id).await
     }
 
-    /// Ban a [`User`] from the guild, deleting a number of days' worth of messages (`dmd`) between
-    /// the range 0 and 7.
+    /// Ban a [`User`] from the guild, deleting a number of seconds' worth of messages (`dms`)
+    /// between the range 0 and 604800 (subject to change). To ban someone for X days', multiply
```

Comment:
```suggestion
    /// `dms` should be between 0 and 604800 (subject to change). To ban someone for X days, multiply
```

Also is "subject to change" here really valid? If anything it would be Discord changing it.